Documentation/colombia_schema.md

```diff
@@ -0,0 +1,30 @@
+
+## schema = colombia
+
+In the schema `colombia` we have datasets related to this country. Up to now, the datasets available are related to the COVID-19 and stratified by departamento<sup>1</sup>. They were provided by the **Instituto Nacional de Salud**. In the table below, we have a brief description of the content of each dataset. For each of them, we have another dataset named `{dataset_name}_meta` that contains an explanation of the columns in the dataset.
```

Comment:
Change th beginning to:
"In the schema `colombia` we currently have 3 datasets related to this country."